packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -186,18 +184,18 @@ export class Expression
         }
     };
 
+    /**
+     * TODO: remove this when everything is pulling from Renderer state
+     * @deprecated get user input from Renderer state
+     */
     getUserInput(): PerseusExpressionUserInput {
-        return normalizeTex(this.props.value);
+        return normalizeTex(this.props.userInput);
     }
 
     getPromptJSON(): ExpressionPromptJSON {
         return _getPromptJSON(this.props, this.getUserInput());
     }
 
-    change: (...args: any) => any | undefined = (...args: any) => {
-        return Changeable.change.apply(this, args);
-    };
```

Comment:
Does this get removed because. you are no longer using onChange?

Reply:
Right, for user input `onChange` has been replaced with `handleUserInput`. For widgets that were only using `onChange` for user input, they no longer need that function/mixin.